src/style.css

```
@@ -0,0 +1,141 @@
+/* #region Normalize */
+
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+
+  --link-height: 40px;
+  --color-dark: #060b35;
+}
+
+.page__content {
+  margin: 0;
+}
+
+.title {
+  font-size: 12px;
+  margin: 0;
+}
+
+.phrase {
+  margin: 0;
+  color: #616070;
+}
+
+/* #endregion */
+
+/* #region card */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
```

you can use `auto` here
```suggestion
  margin: 0 auto 40px;
```